packages/scandipwa/src/component/CheckoutShipping/CheckoutShipping.container.js

```diff
@@ -106,26 +107,19 @@ export class CheckoutShippingContainer extends PureComponent {
     }
 
     componentDidUpdate(prevProps) {
-        const { shippingMethods: prevShippingMethods } = prevProps;
-        const { shippingMethods } = this.props;
+        const { isPickInStoreMethodSelected, shippingMethods = [] } = this.props;
+        const { isPickInStoreMethodSelected: prevIsPickInStoreMethodSelected } = prevProps;
 
-        if (prevShippingMethods !== shippingMethods) {
-            this.resetShippingMethod();
-        }
-    }
-
-    resetShippingMethod() {
-        const { selectedShippingMethod: { method_code: selectedMethodCode = '' } } = this.state;
-        const { shippingMethods } = this.props;
+        const instore = shippingMethods.find((el) => el.method_code === 'pickup');
```

Comment:
Please put strings to constants and store them in the *.conf.js files.

Reply:
done